starter_code/src/movies.js

```diff
@@ -3,10 +3,27 @@
 
 
 // Get the average of all rates with 2 decimals 
+function ratesAverage(array) {
+  var sumOfRatings=array.reduce(function(acc, item){
+    var rateNumber=item.rate*1;    
+    return acc+rateNumber;
+  },0)
 
+  return parseFloat((sumOfRatings/array.length).toFixed(2));
```

Comment:
You can use different ways to parse a string to number. Like `parseFloat()` or `Number()` or ` * 1` or `/ 1`.
Either way, be consistent and choose one.